.github/workflows/main.yml

```diff
@@ -50,9 +50,11 @@ jobs:
       - name: Add Tools to PATH
         run: |
           echo "`pwd`/output/bin" >> $GITHUB_PATH
-      # - name: Install Carrier
-      #   run: |
-      #     ./cli/dist/carrier install
+      - name: Check traefik service
+        run: kubectl get service -n kube-system traefik -o jsonpath='{.status.loadBalancer.ingress[0].ip}'
+      - name: Install Carrier
+        run: |
+          ./cli/dist/carrier install --system-domain "$(kubectl get service -n kube-system traefik -o jsonpath='{.status.loadBalancer.ingress[0].ip}').nip.io"
```

Comment:
I think this somehow got lost - in the initial implementation on kubecfctl we used to generate this if not provided by the user, why not doing the same here?

Reply:
For reference: https://github.com/mudler/kubecfctl/blob/master/pkg/kubernetes/installer.go#L31

Reply:
Note #24

Reply:
Ah! (Did not see the reference in the browser, only after refresh).
I will have a look if that can be added also.

Reply:
Done with commits 118083a58150f52e11e39395b8b1f81a6a94d66d, ceeb8e1be9dc4c70db57e8318037dd65f78d20d3, and eb6d311ad65607705e12870fdcfdb76aab4be329